source/common/http/http2/codec_impl.h

```diff
@@ -400,6 +398,20 @@ class ConnectionImpl : public virtual Connection,
     void grantPeerAdditionalStreamWindow();
   };
 
+  // Encapsulates the logic for sending DATA frames on a given stream.
+  class StreamDataFrameSource : public http2::adapter::DataFrameSource {
+  public:
+    explicit StreamDataFrameSource(StreamImpl& stream) : stream_(stream) {}
+
+    std::pair<int64_t, bool> SelectPayloadLength(size_t max_length) override;
+    bool Send(absl::string_view frame_header, size_t payload_length) override;
+    bool send_fin() const override { return send_fin_; }
```

Comment:
method name not consistent with existing style

Reply:
It's an implementation of an interface from QUICHE code, so changing the method name is not possible in this PR.